test/e2e/network/network_policy.go

```diff
@@ -171,6 +226,85 @@ var _ = SIGDescribe("NetworkPolicy", func() {
 			testCanConnect(f, nsB, "client-b", service, 80)
 		})
 
+		It("should enforce policy, to allow traffic only from a pod in a different namespaces than that of server's namespace,  based on PodSelector and NamespaceSelector [Feature:NetworkPolicy]", func() {
```

Comment:
the descriptions don't need to be so wordy. I'd just say `It("should enforce policy based on combined NamespaceSelector and PodSelector"`. (Likewise with most of the other new tests; follow the example of the existing tests.)

Reply:
I have shortened the description and tried to avoid duplicate descriptions.